lib/core/db_models/helpers/enum_helpers.dart

```diff
@@ -1,12 +1,35 @@
 import 'package:fielder_models/core/constants/app_colors.dart';
 import 'package:fielder_models/core/constants/app_strings.dart';
+import 'package:fielder_models/core/db_models/old/schema/staff_status_schema.dart';
 import 'package:fielder_models/core/enums/enums.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 
 
 class EnumHelpers {
 
+  static CsvWorkerType workerTypeFromString(String type) {
+    switch (type) {
+      case UploadCsvSchema.myFielder:
+        return CsvWorkerType.Fielder;
+      case UploadCsvSchema.myStaff:
+        return CsvWorkerType.Staff;
+      default:
+        return null;
+    }
+  }
+
+  static String stringFromWorkerType(CsvWorkerType type) {
+    switch (type) {
+      case CsvWorkerType.Fielder:
+        return UploadCsvSchema.myFielder;
+      case CsvWorkerType.Staff:
+        return UploadCsvSchema.myStaff;
+      default:
+        return "";
+    }
+  }
+
   static OfferStatus getOfferStatusFromString(String status) {
     switch (status) {
       case 'Pending':
```

Comment:
its not done by you but still if you can change to constant would be great

Reply:
This will be done in a new refactoring task